### PR TITLE
Poc primary font styled component

### PR DESCRIPTION
**Pull request by JackHowa**
poc for reducing unnecessarily duplicated styles

### CODE REVIEW

File: blocks/headline-block/features/headline/headline.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    then render null
+  */
+  (headlineString !== '') && (
+    <PrimaryFont
+      as="h1"
+      className="headline"
+      fontColor="primary-color"
+      // dangerouslySetInnerHTML seems to be a pattern for blocks
+      dangerouslySetInnerHTML={{ __html: headlineString }}
+    />
```
Comment (JackHowa) [inline blocks/headline-block/features/headline/headline.jsx:26]: this is awesome ... using styled components how they were meant to

File: blocks/shared-styles/_children/v-space.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styled from 'styled-components';
+import { framework } from '@wpmedia/news-theme-css/js/framework';
+
+const VSpaceStyles = styled.div.attrs((props) => ({
```
Comment (JackHowa) [inline blocks/shared-styles/_children/v-space.jsx:5]: is this used a lot? I haven't been in the blocks in a while but I'd be curious

File: blocks/shared-styles/index.story.jsx [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    Primary: 'primary-color',
+    Secondary: 'secondary-color',
+  };
+  const defaultValue = 'None';
+  const data = {
+    as: text('"as"', 'h2'),
+    fontColor: select(label, options, defaultValue),
+  };
+
+  const textString = text('Text', 'When this baby hits 88mph...');
```
Comment (JackHowa) [inline blocks/shared-styles/index.story.jsx:46]: heheh

File: .storybook/alias/themes.js [hunk 4]
```diff
@@ -1,5 +1,7 @@
 const themes = () => ({
-	primaryFont: ''
+	'primary-font-family': 'Arial',
```
Comment (JackHowa) [inline .storybook/alias/themes.js:2]: going back to our aliases, I see?

**Comment by matthewroach** [comment]
@JackHowa I am going to close this out in favour of #722 and then we can revisit a vertical spacing component

**Pull request closed without merging**